Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp

```diff
@@ -709,13 +709,17 @@ void Mesh_3_plugin::mesh_3(const Mesh_type mesh_type,
       return;
     }
 #ifdef CGAL_USE_ITK
-    if (sigma_weights > 0 && nullptr == image_item->image_weights())
+    if ( sigma_weights > 0
+      && sigma_weights != image_item->sigma_weights())
     {
-      image_item->set_image_weights(std::move(
-          CGAL::Mesh_3::generate_weights(*pImage, sigma_weights)));
+      image_item->set_image_weights(
+        std::move(CGAL::Mesh_3::generate_weights(*pImage, sigma_weights)),
```

Comment:
```suggestion
        CGAL::Mesh_3::generate_weights(*pImage, sigma_weights),
```
`generate_weights` return a value, a temporary without any name, so `std::move` is useless because the temporary is already a rvalue-reference (a movable object).

Reply:
right, I'm changing it!